opencti-platform/opencti-graphql/src/initialization.js

```diff
@@ -253,10 +253,9 @@ export const createBasicRolesAndCapabilities = async (context) => {
   await createCapabilities(context, CAPABILITIES);
   // Create roles
   await addRole(context, SYSTEM_USER, {
-    name: ROLE_DEFAULT,
-    description: 'Default role associated to all users',
+    name: ROLE_KNOWLEDGE,
```

Comment:
why this change? to be clearer on the naming of the roles?

Reply:
yes, it's because it's no more a default role, but a role with knowledge capabilities